### PR TITLE
Add a standard subscription [ci-skip]

### PR DESCRIPTION
**Pull request by ColinSullivan1**
Add a service subject that isn't load balanced for example/demonstration purposes.

Signed-off-by: Colin Sullivan <colin@synadia.com>

### CODE REVIEW

File: examples/nats-echo/main.go [hunk 1]
```diff
@@ -115,6 +115,14 @@ func main() {
 				nc.Publish(msg.Reply, msg.Data)
 			}
 		}
+	}
+
+	nc.QueueSubscribe(subj, "echo", func(msg *nats.Msg) {
+		handleMsg(msg)
+	})
+	allSubj := subj + ".all"
```
Comment (derekcollison) [inline examples/nats-echo/main.go:123]: Let's make this status.

Comment (derekcollison) [reply examples/nats-echo/main.go:123]: And for this one we just should send back our geo I think? Maybe if you want add in an identifier at startup to customize?

**Review by derekcollison — CHANGES_REQUESTED** [review]

File: examples/nats-echo/main.go (hunk 1 above)
Comment (ColinSullivan1) [reply examples/nats-echo/main.go]: Fixed; it's a status and there is an identifier now.

**Comment by tbeets** [comment]
LGTM

**Review by tbeets — APPROVED** [review]
LGTM

**Review by philpennock — APPROVED** [review]

**Pull request merged**